package.json

```diff
@@ -24,6 +24,12 @@
     "type": "git",
     "url": "git://github.com/js-cookie/js-cookie.git"
   },
+  "files": [
+    "src/**/*.js",
+    "MIT-LICENSE.txt",
+    "SERVER_SIDE.md",
+    "CONTRIBUTING.md"
```

Comment:
[Here](https://docs.npmjs.com/files/package.json#files) they say that npm inspects "CHANGELOG (and its variants)", maybe we could find which ones are the "variants" and see if they work nicely with Github and NPM?